decidim-proposals/app/services/decidim/proposals/proposal_serializer.rb

```diff
@@ -37,7 +43,16 @@ def organization
         feature.organization
       end
 
-      attr_reader :proposal
+      def meetings
+        @proposal.linked_resources(:meetings, "proposals_from_meeting").map do |meeting|
+          Decidim::Meetings::ListEngine.routes.url_helpers.meeting_url(
```

Comment:
This is creating a hard dependency on the meetings engine, can we used `decidim_resource_url`?